src/components/message-list/_macro.spec.js

```diff
@@ -39,7 +40,11 @@ const EXAMPLE_MESSAGE_LIST = {
     ...EXAMPLE_MESSAGE_LIST_MINIMAL,
     ariaLabel: 'Message list for ONS Business Surveys',
     ariaLabelMetaData: 'Message information',
-    ariaLabelMsg: 'Message preview',
+};
+
+const EXAMPLE_MESSAGE_LIST_WITH_DEPRECATED_ariaLabelMsg_PARAM = {
+    ...EXAMPLE_MESSAGE_LIST_MINIMAL,
+    ariaLabelMsg: 'Message Preview',
 };
```

Comment:
This doesnt need to be its own example config as its only being used in one test at the moment you can just add the param in that test. Also the name needs updating because it should be all in caps